### PR TITLE
Darlp networks 1202309411800128

### PR DESCRIPTION
**Pull request by darlp5589**
ASANA Ticket source: https://app.asana.com/0/1201943683850147/1202309411800128

@chriscodev , kindly review this PR. This push commit is limited only for BIG COMMIT "added" array features below:
-newly added network (or set of networks)
-newly added network (or set of networks) with one or more channels
-newly added network (or set of networks) with one or more channels with or without transport links
-auto-refresh after network to latest set of networks after commit

Big COMMIT for "updated" array will be in the next push commit.

### CODE REVIEW

**Review by chriscodev — CHANGES_REQUESTED** [review]
- adding atsc chanel model title is edit
- change input field layout to be consistent with other input fields
- 
- Screenshot 2022-05-26 220310

- change input field layouts

**Review by chriscodev — COMMENTED** [review]
This is approve commit: https://github.com/triveni-codev/GBWebApp-dev/pull/432/commits/053a69fc3958b667c3beccd515031eab4011f031

to note: implementations are commented out or put as hidden for now, please know that this PR is currently under development and that I need those implementation as I currently working on some features included in the scope of this ticket.

**Review by chriscodev — COMMENTED** [review]
hi Darl , here are my review comments:
- to be same layout as modulation select tag
-  to add default value 'Select Timezone' placeholder same in client app
-  same width with linked transport
- add placeholder to link transport same in client app
atsc

**Comment by chriscodev** [comment]
hi @darlp5589 , 

COMMIT : https://github.com/triveni-codev/GBWebApp-dev/pull/432/commits/6fe0ac61cf90f2a356fb6cf6a46d653303a3a2c5

approve on my end.

Cheers,
Christian

File: src/app/_helpers/idClientIdParam.ts [hunk 1]
```diff
@@ -1,7 +1,12 @@
-export function addIdClientId(data, btnAction, id, clientId) {
+import { min } from "moment";
```
Comment (chriscodev) [inline src/app/_helpers/idClientIdParam.ts:1]: not used, we can remove this one.

**Review by chriscodev — COMMENTED** [review]
not used, we can remove lines codes.

File: src/app/_helpers/idClientIdParam.ts (hunk 1 above)
Comment (darlp5589) [reply src/app/_helpers/idClientIdParam.ts:1]: Oh yeah! Makes sense! Will do.

**Comment by chriscodev** [comment]
hi @darlp5589 ,

heres my review comment

when adding data then edit ,clientid should be used on add-editing, cause this will get the data of previous data of id, causing to display invalid data

bug2

**Comment by darlp5589** [comment]
Hi @chriscodev , kindly leave this as is. This was Don's implementation for data-passing purposes (I make use of it so I will have few edits to make on his previous implementation) but if you look on the set of objects, there's an associated clientID which is what I am using for the big commit api request.

**Comment by triveni-codev** [comment]
Post-discussion w/ Christian and Darl via call was carried out.
Resolution: Code will be updated to consider the fundamental scenario

Fundamental scenario to consider (in relation to the image in https://github.com/triveni-codev/GBWebApp-dev/pull/432#issuecomment-1153363188)
Reviewer:  Christian
Topic: clashing of information of a newly added id 33 vs an existing id 33
1. Current id info from server for example :39 and 33
2. Added new Transports until id 33 is reached
3. Click on a server-existing Transport row to update(presumably id 33 from server)
Actual result: name from client side is viewed
Expected result: name/info from server should be viewed in the modal
~~~~~~~~~~~
Client id val: Date time value down to millisecond (edited)

**Review by chriscodev — APPROVED** [review]
LGTM

File: src/app/pages/dashboard/network-config/networks/networks.component.ts [hunk 2]
```diff
@@ -252,19 +251,33 @@ export class NetworksComponent implements OnInit {
 							const ipsettings = this.tableNetworksData;
 							let dataRow = ipsettings.find((obj) => obj.id === tableID);							
 							let dataValues = {};
+							if(dataRow === undefined) {
```
[the remark this answers is not in the document]
Comment (darlp5589) [reply src/app/pages/dashboard/network-config/networks/networks.component.ts:254]: Gotcha!

**Comment by triveni-codev** [comment]
Scoped out and reviewed by Christian. Merging this PR. I may have newfound "dynamics" concern leading up to succeeding PRs. previous prevailing concern on the modified  "min.scss" under eliteadmin assets folder has been addressed and sorted out.

**Pull request merged**